### DISCUSSION TITLE
AtomRCF405navi_DELUX.hex? What's different from the original non-DELUX?

### DISCUSSION THREAD

**Discussion by FlapsLafferty** [iNavFlight/inav#10315] [Q&A]
I am using an AtomRCF405navi FC in a Swordfish plane with iNav 7.1.2. Noticed a new, 2nd listing in the targets for a "DELUX" version.
Does anyone have knowledge about this target?  Is it safe to load into a 1-year old FC?

**Comment by b14ckyy** [comment]
probably a different variant of the FC with other components. Please only use targets that are made for your specific FC.

**Reply by FlapsLafferty** [reply]
b14ckyy,
Thanks for your reply. Makes sense. I'm always looking for ways to optimize my rig. The AtomRC.com website does not sell a FC with that target. They show the same target for both the mini and std size of their F405 FC. Will keep my eye on the market for the DELUX FC and its specs.

**Reply by b14ckyy** [reply]
who knows. Pawel added it 9 months ago. But he is not available at the moment to ask. maybe they cancelled the FC in the end? No idea.
